src/oracle/connection/oracle_value.rs

```diff
@@ -9,12 +9,11 @@ pub struct OracleValue<'a> {
 pub(crate) enum InnerValue<'a> {
     Raw {
         raw_value: &'a oracle::SqlValue,
-        tpe: &'a oracle::sql_type::OracleType,
+        tpe: oracle::sql_type::OracleType,
     },
     SmallInt(i16),
     Integer(i32),
     BigInt(i64),
-    Bool(bool),
```

Comment:
Is it ok that `bool` is gone now?

Reply:
It is not needed anymore as we use `SmallInt` internally for this now (https://github.com/GiGainfosystems/diesel-oci/pull/41/files/2748b78bc2148be11b39d0699ebe25d2869f0cd9#diff-e6df06b79f25b6537cfe851fe8c707084197677482e0edf94f74655b4baa3f5aR359) and handle the conversion directly in the corresponding `FromSql` impl: https://github.com/GiGainfosystems/diesel-oci/pull/41#diff-08866ee1105680dbb2a25006ffd51ffb5b98b7358c1752d2cbbec40a0fa5d4eaR169